keras/engine/training.py

```diff
@@ -1442,57 +1442,59 @@ def fit(self, x=None,
         """Trains the model for a fixed number of epochs (iterations on a dataset).
 
         # Arguments
-            x: Numpy array of training data,
-                or list of Numpy arrays if the model has multiple inputs.
-                If all inputs in the model are named,
-                you can also pass a dictionary
-                mapping input names to Numpy arrays.
-                Can be `None` (default) if feeding from framework-native tensors.
-            y: Numpy array of target data,
-                or list of Numpy arrays if the model has multiple outputs.
-                If all outputs in the model are named,
-                you can also pass a dictionary
-                mapping output names to Numpy arrays.
-                Can be `None` (default) if feeding from framework-native tensors.
+            x: Numpy array of training data, or 1-element list of Numpy arrays.
```

Comment:
It can be a list, in case of multi-input models; does not have to be a 1-element list